### PR TITLE
DOC: Updating See Also section in IndexOpsMixin

### PR DESCRIPTION
**Pull request by Sedosa**
- [x] Updated the "See Also" section for the IndexOpsMixin '''_is_homogenous_type'''

### CODE REVIEW

File: pandas/core/base.py [hunk 1]
```diff
@@ -678,8 +678,10 @@ def _is_homogeneous_type(self):
 
         See Also
         --------
-        DataFrame._is_homogeneous_type
-        MultiIndex._is_homogeneous_type
+        DataFrame._is_homogeneous_type : Whether all the columns in a
+            DataFrame have the same dtype.
+        MultiIndex._is_homogeneous_type : Whether all the levels of a
+            MultiIndex  have the same dtype.
```
Comment (jschendel) [inline pandas/core/base.py]: there's an extra space between "MultiIndex" and "have"

**Comment by jschendel** [comment]
Thanks!

Can you also update the See Also section for `MultiIndex._is_homogeneous_type` in a similar manner:
https://github.com/pandas-dev/pandas/blob/b1c3a9031569334cafc4e8d45d35408421f7dea4/pandas/core/indexes/multi.py#L666-L669

And can you add a See Also section to `DataFrame._is_homogeneous_type`?  It should essentially be a copy/paste combining the Index/MultiIndex entries from the other See Also sections:
https://github.com/pandas-dev/pandas/blob/b1c3a9031569334cafc4e8d45d35408421f7dea4/pandas/core/frame.py#L530-L552

File: pandas/core/base.py (hunk 1 above)
Comment (Sedosa) [reply pandas/core/base.py]: Amended

**Comment by Sedosa** [comment]
The changes have been added @jschendel

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @Sedosa

**Pull request merged**